src/pages/images/ImageSelector.tsx

```diff
@@ -260,7 +261,12 @@ const ImageSelector: FC<Props> = ({ onSelect, onClose }) => {
             onClick: selectImage,
           },
           {
-            content: getSource(),
+            content: (
+              <>
+                {getSource()}{" "}
+                {item.cached && <span className="u-text--muted">cached</span>}
```

Comment:
The space should be inside the conditional.

Reply:
thanks! didn't pick that up as "cached" were all on a new line for me